unittests/scans/csaf/csaf_one_vuln.json

```diff
@@ -0,0 +1,63 @@
+{
+  "document": {
+    "category": "csaf_security_advisory",
+    "csaf_version": "2.0",
+    "publisher": {
+      "category": "user",
+      "name": "test_publisher",
+      "namespace": "http://www.foo.de"
+    },
+    "title": "A very serious vulnerability for testing",
+    "tracking": {
+      "current_release_date": "2022-06-01T10:00:00.000Z",
+      "id": "Example Vulnerability - Testing Company",
```

Comment:
Please review [section 5.1](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#51-filename) of the CSAF standard to generate the correct filename (or adopt the `id` from the filename). (Applies to all.)